.ci/docker/common/install_cache.sh

```diff
@@ -12,6 +12,30 @@ set -ex
 # shellcheck source=/dev/null
 source "$(dirname "${BASH_SOURCE[0]}")/utils.sh"
 
+install_ubuntu() {
+  echo "Preparing to build sccache from source"
+  apt-get update
+  # libssl-dev will not work as it is upgraded to libssl3 in Ubuntu-22.04.
+  # Instead use lib and headers from OpenSSL1.1 installed in `install_openssl.sh``
+  apt-get install -y cargo
+  echo "Checking out sccache repo"
+  if [ -n "$CUDA_VERSION" ]; then
```

Comment:
I think we can just use upstream https://github.com/mozilla/sccache to keep it simple here as there is no CUDA